COMPILING.md

```diff
@@ -45,6 +45,7 @@ For Linux, Mac and MinGW:
 $ git clone https://github.com/google/protobuf.git
 $ cd protobuf
 $ git checkout v3.9.0
+$ git submodule update --init --recursive
```

Comment:
You shouldn't need the submodules. I normally do a build from the .tar.gz, which also doesn't have submodules.

Hmm... we also run "make check"? Meh. And autogen shouldn't be necessary any more.

Let me send out some modifications here.

Reply:
For the "make check" to succeed we need submodules - that's what I ran into

Reply:
Sent out #6249, which I tested on a fresh Debian Buster container.

Reply:
Ah, I see. That makes sense, because that wasn't likely previously necessary. I think it's better to just remove the make check.

Reply:
removed this update from the PR.